### DISCUSSION TITLE
Gmail stopped supporting app passwords for new accounts

### DISCUSSION THREAD

**Discussion by pchiquit** [moralmunky/Home-Assistant-Mail-And-Packages#756] [Q&A]
Google says this:
`To help keep your account secure, from May 30, 2022, ​​Google no longer supports the use of third-party apps or devices which ask you to sign in to your Google Account using only your username and password.`

On the Account settings -> Security where that would be available it now says:
![image](https://user-images.githubusercontent.com/24482951/203694891-cbab782a-4656-4e07-acda-a83d02eb8fdd.png)

**Comment by firstof9** [comment]
OAuth is still a work in progress.
